algorithms/alg.verify-tx.tex

```diff
@@ -0,0 +1,19 @@
+\begin{algorithm}[H]
+    \caption{\label{alg.verify-tx} A function to verify that a transaction is conforming to the provided event and is included in the provided block.}
+    \begin{algorithmic}[1]
+        \Function{\sf verify-tx$_e$}{$\alpha$}
+            \Let{tx}{\textsf{decodeRawTX}(\alpha.\textsf{txRaw})}
```

Comment:
I'd destructure $\alpha$ into \txRaw etc.

Reply:
Define \txRaw macro.

Reply:
I feel the decoding/encoding into strings that you're doing here is an engineering issue which is irrelevant in the scientific treatment. Traditionally, in computer science we generally assume that the encoding of an object is the same as the object itself, unless these two are materially different and this plays a role in the representation. As such, I feel these algorithms could be better written if this distinction is not made and remains abstract. Hence, \alpha can contain a \tx and a \blk, not a \txRaw and \blockHeader. The difference doesn't seem to matter for the context of this paper.